internal/controller/auth.controller.go

```diff
@@ -177,7 +177,7 @@ func (ac *AuthController) ResetPassword(ctx *gin.Context) {
 // @Accept json
 // @Produce json
 // @Param data body auth_dto.ChangePasswordInput true "data"
-// @Router /reset-password [put]
+// @Router /change-password [put]
```

Comment:
Did you change the path in auth router ?

Reply:
Yes i did. I checked on Swagger UI and recognized it's wrong so i fixed that.
![image](https://github.com/user-attachments/assets/4174472a-d1c1-4af1-9b59-7fad22601eea)